%YAML:1.0
A_hdcamera: !!opencv-matrix
 rows: 3
 cols: 3
 dt: d
 data: [ 1381.674685379492, 0.0, 1004.532889907346, 0.0, 1380.876638649972, 561.275900198695, 0.0, 0.0, 1.0 ]

A_kinect: !!opencv-matrix
 rows: 3
 cols: 3
 dt: d
 data: [ 528.5975091108814, 0.0, 345.7334483645757, 0.0, 527.1975760998138, 264.9890958167526, 0.0, 0.0, 1.0 ]

M_extrinsic_kinect_to_HD: !!opencv-matrix
 rows: 3
 cols: 4
 dt: d
 data: [ 0.997419041165492,  -0.014195796394487,   0.070382779749283, 2.261259544127428, 0.010312159890845,  0.998419949188335,   0.055238251430940, 47.654476751874711, -0.071055722351514,  -0.054369885299570,   0.995989457722127, 43.396987341785504]

M_extrinsic_HD_to_kinect: !!opencv-matrix
 rows: 3
 cols: 4
 dt: d
 data: [0.9974,    0.0103,   -0.0711 ,  0.3367, -0.0142,    0.9984,   -0.0544, -45.1875, 0.0704,    0.0552,   0.9960, -46.0145]

